### ISSUE TITLE
does not help when root or swap partition is encrypted

### ISSUE THREAD

**Issue by vsespb** [DarkCaster/Systemd-Issue1620-Workaround#1]
and it's not clear from the docs.

**Comment by DarkCaster** [comment]
Yes, it is not able to deal with root and swap partitions for now. I'll try to add support for encrypted swap partitions soon. But root partition handling is a task for initrd, so there is no easy way to make universal workaround for root partition.

**Comment by ghost** [comment]
I just installed this in the hope it would help with dismounting encryted swap. I'm pretty disappointed to discover that it does not even try!

**Comment by vsespb** [comment]
i think that's not a problem that it does not work with encrypted swap. problem that it's not documented.
